rfc/src/template.md

```diff
@@ -10,47 +10,66 @@
 
 ## Summary
 
-Short description of the feature, i.e.: the elevator pitch. What is this feature about?
+Short (1-2 sentences) description of the feature. What is this feature about?
 
 ## User Impact
 
 Why are we doing this? How will this benefit the final user?
 
  - If this is an API change, how will that impact current users?
- - For deprecation or breaking changes, how will the transition look like?
  - If this RFC is related to change in the architecture without major user impact, think about the long term
-impact for user. I.e.: what future work will this enable.
+impact for user. I.e.: what future work will this enable. Maybe reach out to other developers to ensure you  need an RFC in the first place.
```

Comment:
Double space.

And I would say "If you are unsure, reach out to the Kani developers ..." and perhaps even clarify that that "reaching out" should probably be a feature request issue.